src/inputs/dropdown/dropdown.jsx

```diff
@@ -157,6 +163,8 @@ class Dropdown extends React.Component {
                     const value = {
                         id: itemId || null,
                         label: itemLabel,
+                        onClick,
```

Comment:
This changes was included to get the <DropDownItem /> value on the <DropDown /> onChange. (It seems that we can set the value property for each item but we're getting the selected option only with id and label).
`http://cm-ui-docs.saddleback.com/components/inputs/dropdown-deprecated#disable`

Reply:
We should be using Dropdown Button.

Reply:
Done